.devcontainer/Dockerfile

```diff
@@ -13,4 +11,4 @@ FROM mcr.microsoft.com/vscode/devcontainers/typescript-node:0-${VARIANT}
 # RUN su node -c "source /usr/local/share/nvm/nvm.sh && nvm install ${EXTRA_NODE_VERSION}"
 
 # To install more global node packages
-RUN su node -c "npm install -g pnpm ts-node"
+RUN su node -c "npm install -g pnpm@6 ts-node"
```

Comment:
Should we also pin ts-node version, just in case?

Reply:
We didn't encounter issues with ts-node major so far, maybe we wait and see if it's needed